aquadoggo/src/db/models/log.rs

```diff
@@ -93,12 +93,12 @@ impl Log {
         Ok(next_log_id)
     }
 
-    /// Returns the registered log_id for a document.
+    /// Returns the registered log_id for a document identified by its hash.
     ///
-    /// Operations are separated in different logs per document and author. This method checks if a log
-    /// has already been registered for a document and returns its id.
-    pub async fn get(pool: &Pool, author: &Author, document: &Hash) -> Result<Option<LogId>> {
-        // @TODO: Look up if system schema was used and return regarding log id
+    /// Operations are separated in different logs per document and author. This method checks if a
+    /// log has already been registered for a document and author and returns its regarding log id
+    /// or None.
+    pub async fn get(pool: &Pool, author: &Author, document_hash: &Hash) -> Result<Option<LogId>> {
```

Comment:
Following on from our absurd id naming jam... It's places like this where we could do with a naming convention. Just wondering how it would be reading this code relatively "cold", I would probably think `document_hash` was the actual hash of this document. Maybe whenever we are using a hash as an id we call it `document_hash_id` and when whenever it's the actual hash of that item, we call it `name_hash`?

Open to any other suggestions too though....

Reply:
I like that, good idea!